src/validators/validators.js

```diff
@@ -11,7 +12,10 @@ const titleSpecialCharCheck = ({ title, isFile = false }) => {
     // Remove .md
     testTitle = title.replace(/.md$/, "")
   }
-  return specialCharactersRegexTest.test(testTitle)
+  return (
+    specialCharactersRegexTest.test(testTitle) ||
+    jekyllFirstCharacterRegexTest.test(testTitle)
```

Comment:
Hi shouldnt this be a '&&'  rather than a "||"?

Reply:
`||` is correct - this method is checking for the existence of any invalid characters, but I think the method name is confusing about this. I've renamed the method to `hasSpecialCharInTitle` in https://github.com/isomerpages/isomercms-backend/pull/699/commits/b8a41a3b1937f5aa2852f58780fed2d4e660d99f